src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -466,9 +484,20 @@ else if (SiteUtil.JAVADOC_TOKENS.equals(propertyName)) {
         else {
             final String defaultValue = SiteUtil.getDefaultValue(
                     propertyName, field, instance, currentModuleName);
-            sink.rawText(CODE_START);
-            sink.text(defaultValue);
-            sink.rawText(CODE_END);
+
+            final boolean isSpecialTokenProp = NON_BASE_TOKEN_PROPERTIES.stream()
+                    .anyMatch(tokenProp -> tokenProp.endsWith(" - " + propertyName));
```

Comment:
This isn't safe code. You are matching by property name and completely ignoring the check name. This will cause issues if different checks use the same property name.

Reply:
Thank you for pointing it out.

Reply:
@rnveach , this item is addressed.